.github/workflows/ci.yml

```diff
@@ -2,7 +2,6 @@ name: ci
 
 on:
   push:
-  pull_request:
```

Comment:
We don't need this on `pull_request` if we are running on every push.
https://github.com/orgs/community/discussions/25654